__tests__/identity.test.js

```diff
@@ -267,7 +276,9 @@ describe('Tozny identity client', () => {
     }
     const response = await ops.approveAccessRequests(
       realmConfig,
-      identity,
+      // you can't approve your own requests so a different identity must perform approval.
```

Comment:
this is a desired feature though and would immediately make everyone's life at Tozny easier that has to go through mpc flows for things that we can trust (but want to audit) tozny engineers to be able to self approve for, I would strongly consider "just be a hero (a super hero)" if adding support for this feature is less than a days work.